---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

## WORK EXPERIENCE

### Senior Bioinformatician (Operations)

**Jan. 2023 - Present**

_Genomics England, London, UK_

- Leading development of a new quality control monitoring system for pipeline runs using AWS systems and NextFlow Tower.
- Increased automation and security in team CI/CD pipeline, including developing a cookiecutter template for initialising future repositories.
- Line manage 1 junior member of Operations team, providing technical and professional mentorship with regular one-on-ones and semi-annual performance reviews.
- Led the team through both ISO 13485 and ISO15189 audits that returned no findings, ensuring team SOPs were up-to-date and answering auditor questions.
- Performed gap analysis to ensure team remained compliant with new ISO 15189:2022 standards.
- Represent the team at meetings with other squads, serving as a subject matter expert for company decisions that impact pipeline operations.

### Bioinformatician (Operations)

**June 2022 - Jan. 2023**

_Genomics England, London, UK_

- Took initiative to improve documentation of team codebase by standardising docstrings across the repository to auto-build HTML documentation with Sphinx.
- Lead Operations support of Research Cohorts, creating a clearly defined protocol for testing in UAT and facilitating collaboration across teams to remove testing and production roadblocks.
- Ensured smooth operation of HPC pipeline by overseeing workflows to monitor for failures, intervening when necessary to apply ISO-certified fixes.
- Robust coding and teamwork skills refined by regular contribution and review of pull requests to implement process improvements.

### Computational Biologist

**Nov. 2020 - June 2022**

_Bioinformatics & Biostatistics (BIO2) Core, Institute for Metabolic Sciences, University of Cambridge, UK_

_Merkle Lab, Institute for Metabolic Sciences, University. of Cambridge, UK_

- Provide expert analysis for key stakeholders to deliver critical recommendations that guide decision-making for future strategy.
- Wrote and implemented the Merkle Lab’s FAIR/O compliant data management policy to take advantage of remote storage and automation, which reduced overhead for wet-lab colleagues.
- Led the initiative to write the BIO2 Core’s external collaboration policy to guide future projects and ensure successful outcomes.
- Developed LipidTA, an open-source Python command line interface built to translate complex lipidomics data into meaningful insights.
- Constructed self-documenting, open-source pipelines for data analysis tasks ranging from external API queries to producing reports on diverse data anylses.

### Computational Biologist

**Oct. 2019 - Oct. 2020**

_National Institute of Neurological Disorders and Stroke, NIH, Bethesda, Maryland, USA_

- Developed a data-driven framework, ChooseR, in R that used a novel metric of cluster robustness to provide a quantitative measure of the quality of chosen clustering parameters. Led to a first-author publication in BMC Bioinformatics.
- Integrated diverse datasets to produce a reference atlas of spinal cord cell types and subsequently developed a machine learning pipeline to use this atlas to classify new cells. Led to a publication in Nature Communications.
- Worked in a team of wet-lab scientists to translate experimental data into meaningful interpretations and actionable recommendations. Regularly communicated these analyses to team members.

### Masters Candidate

**Sept. 2015 - Oct. 2019**

_Dept. of Physiology, Anatomy, and Genetics, University of Oxford, UK_

_National Institute of Neurological Disorders and Stroke, NIH, Bethesda, Maryland, USA_

- Constructed applet in ImageJ Macro Language, a derivative of Java, to improve high-throughput analysis of super-resolution fluorescence microscopy data.
- Wove four years of experiments and data analysis into a cohesive story providing meaningful insight into the effects of autophagic perturbation on mitochondrial function in my thesis
- Strong organizational, teamwork, and collaboration skills expanded by managing an international, collaborative research project.

### Dept. of Neuroscience, Johns Hopkins University, Baltimore, MD, USA

_Research Assistant_, Sept. 2011 - May 2015

- Supervisor: Prof. Hey-Kyoung Lee
- Assisted in experiments examining the effects of visual deprivation on the strength of synapses in the primary auditory and visual cortices of mice that led to a Journal of Neuroscience publication.

## TEACHING EXPERIENCE

  <ul>{% for post in site.teaching %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>

## EDUCATION

### MSc by Research in Medical Science

**March 2020**

_Universityof Oxford, UK_

- Thesis: _Investigating the effects of autophagic perturbations on mitochondrial function in Parkinson's Disease_

### Foundation for Advanced Education in the Sciences, Bethesda, MD, USA

_BIOF399_, Deep Learning for Healthcare Image Analysis, Sept. - Dec. 2019

- Applications of convolutional neural networks (CNNs) to data to create GPU accelerated networks using Keras and TensorFlow.

_BIOF509_, Applied Machine Learning, Feb. - May 2019

- Broad, practical overview of machine learning, covering regression, random forest, K-means clustering, and deep learning using scikit-learn and tensorflow.

_BIOF518_, Theoretical and Applied Bioinformatics, Sept. - Dec. 2018

- Covering the theory and practice of a wide range of bioinformatic techniques, including BLAST, gene structure prediction, phylogenetics, and sequence alignments.

_BIOF309_, Introduction to Python, Sept. - Dec. 2018

- Developing core Python principles such as syntax, reading/writing files, debugging, regular expressions, and creating modules.

_[Scientists Teaching Science](https://www.training.nih.gov/sts_main_page)_, Mar. - Apr. 2018

- Teaching course for scientists that developed principles of effective teaching and communication, including enquiry-based learning, course planning, and exam writing.

### BSc in Neuroscience and Molecular & Cellular Biology

**May 2015**

_Johns Hopkins University, Baltimore. MD, USA__

- General Honors (GPA: 3.91) - First equivalent

### Minor in Spanish for the Professions

**May 2015**

_Johns Hopkins University, Baltimore. MD, USA__

## Core Skills

### Technical

- Full stack data analysis with Pandas/Numpy/Tidyverse
- Data visualisation with Matplotlib/Seaborn/ggplot2
- App development for data exploration in R (Shiny) and Python (dash)
- Command-line-interface (CLI) tool development in Python and Shell
- Pipeline development using Snakemake
- Continuous integration with Github Actions, Travis CI, CodeCov, PyUp, PyTest, and Read the Docs
- Shell scripting in various dialects (Bash, Zsh, Fish)
- Single cell RNA sequencing analysis in R (Seurat/Scran) and Python (Scanpy/scVI)
- High-throughput data analysis scripts in ImageJ using a derivative of Java
- Software version control with git/GitHub
- Remote, high performance computing experience (AWS)

### Laboratory

- Airy scan microscopy (Zeis 880 platform)
- High-throughput laser confocal microscopy (Perkin Elmer Phenix platform)
- Immunofluorescence
- General cell culture
- Chemical differentiation of iPSCs
- Plasmid and viral vector design and construction
- Western blotting
- DNA gel electrophoresis
- Mouse surgery and dissection
- Biotinylation
- Golgi staining
- Brain slice preparation for electrophysiology

## AWARDS

### Intramural Research Training Award

_National Institutes of Health, Sept 2015 - Jan 2020_

_Bethesda, MD, USA_

### Barry Goldwater Scholarship

_The Barry Goldwater Scholarship and Excellence in Education Foundation, 2014_

## PUBLICATIONS

  <ul>{% for post in site.publications %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
