---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

## WORK EXPERIENCE

### Bioinformatics and Biostatistics Core, Institiute for Metabolic Science, University of Cambridge, UK

_Computational Biologist_, Nov. 2020 - Present

* Provide expert computational support across many projects and teams in a strongly collaborative environment.
* Create pipelines in Snakemake for analyses from scRNAseq to API queries in order to produce FAIR/O projects.
* Developed the Core's collaboration policy for new and incoming projects. 

### The National Institute of Neurological Disorders and Stroke, NIH, Bethesda, MD, USA

_Pre-Doctoral Fellow_, Oct. 2019 - Oct. 2020

* Supervisor: Dr. Ariel Levine
* Developed novel algorithm in R to create an intuitive, quantitative metric of cluster quality to reduce bias during cell-type identification in next-generation sequencing datasets. This work led to one manuscript, published in _BMC Bioinformatics_.
* Implemented multiple machine learning classifiers in Python, including support vector machines and neural networks, to create a cell-type classifying algorithm for implementation with a reference, next generation sequencing dataset. This work led to one manuscript, currently in press at _Nature Communicastions_.
* Full analysis of single cell RNA sequencing datasets, from pre-processing data using Bash scripting and command-line tools, to using R and Python packages, such as Seurat and Scanpy, for clustering and differential expression testing.
* Excellent communication skills honed through frequent collaborations with laboratory colleagues.

### Dept. of Physiology, Anatomy, and Genetics, The University of Oxford, Oxford, UK, and The National Institute of Neurological Disorders and Stroke, NIH, Bethesda, MD, USA

_Masters Candidate_, Sept. 2015 - Oct. 2019

* Supervisors: Prof. Richard Wade-Martins (Oxford) and Dr. Craig Blackstone (NIH)
* Developed assay for measuring mitophagy in situ to perform experiments examining the effects of autophagic perturbations on mitochondrial function.
* Built pipeline for automated high-throughput imaging and analysis of data files greater than 100 Gb using the Perkin Elmer High Content Screening System
* Strong organizational, teamwork, and collaboration skills expanded by managing a research project in labs on 2 continents.
* Exceptional communication skills built through quarterly presentations in lab meeting and 1 invited talk at Master's program colloquium.

### Dept. of Neuroscience, Johns Hopkins University, Baltimore, MD, USA

_Research Assistant_, Sept. 2011 - May 2015

* Supervisor: Prof. Hey-Kyoung Lee
* Assisted in experiments examining the effects of visual deprivation on the strength of synapses in the primary auditory and visual cortices of mice that led to a Journal of Neuroscience publication.

## TEACHING EXPERIENCE

  <ul>{% for post in site.teaching %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>

## EDUCATION

### The University of Oxford, Oxford, UK

_MSc by Research_, Medical Science, Oct. 2019

* Thesis: _Investigating the effects of autophagic perturbations on mitochondrial function in Parkinson's Disease_

### Foundation for Advanced Education in the Sciences, Bethesda, MD, USA

_BIOF399_, Deep Learning for Healthcare Image Analysis, Sept. - Dec. 2019

* Applications of convolutional neural networks (CNNs) to data to create GPU accelerated networks using Keras and TensorFlow.

_BIOF509_, Applied Machine Learning, Feb. - May 2019

* Broad, practical overview of machine learning, covering regression, random forest, K-means clustering, and deep learning using scikit-learn and tensorflow.

_BIOF518_, Theoretical and Applied Bioinformatics, Sept. - Dec. 2018

* Covering the theory and practice of a wide range of bioinformatic techniques, including BLAST, gene structure prediction, phylogenetics, and sequence alignments.

_BIOF309_, Introduction to Python, Sept. - Dec. 2018

* Developing core Python principles such as syntax, reading/writing files, debugging, regular expressions, and creating modules.

_[Scientists Teaching Science](https://www.training.nih.gov/sts_main_page)_, Mar. - Apr. 2018
* Teaching course for scientists that developed principles of effective teaching and communication, including enquiry-based learning, course planning, and exam writing.

### Johns Hopkins University, Baltimore, MD, USA

_BSc_, Neuroscience and Molecular & Cellular Biology, May 2015

* General Honors (GPA: 3.91)

_Minor_, Spanish for the Professions, May 2015

## ADDITIONAL SKILLS

### Data Science

* Single cell RNA sequencing analysis in R (Seurat/Scran) and Python (Scanpy/scVI)
* Pipeline development using Snakemake
* App development for data exploratopn in R (Shiny) and Python (dash)
* Data wrangling with they tidyverse (R) and pandas & numpy (Python)
* Command-line-interface (CLI) tool development in Python
* Continuous integration with Github Actions, Travis CI, CodeCov, PyUp, PyTest, and Read the Docs
* Shell scripting in various dialects (Bash, Zsh, Fish)
* High-throughput data analysis scripts in ImageJ using a derivative of Java
* Software version control with git/GitHub
* Remote, high performance computing experience (AWS)

### Laboratory

* Airy scan microscopy (Zeis 880 platform)
* High-throughput laser confocal microscopy (Perkin Elmer Phenix platform)
* Immunofluorescence
* General cell culture
* Chemical differentiation of iPSCs
* Plasmid and viral vector design and construction
* Western blotting
* DNA gel electrophoresis
* Mouse surgery and dissection
* Biotinylation 
* Golgi staining
* Brain slice preparation for electrophysiology

## AWARDS

### Intramural Research Training Award

_National Institutes of Health, Sept 2015 - Jan 2020_

_Bethesda, MD, USA_

### Barry Goldwater Scholarship

_The Barry Goldwater Scholarship and Excellence in Education Foundation, 2014_

## PUBLICATIONS

  <ul>{% for post in site.publications %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
