---
title: "A Harmonized Atlas of Spinal Cord Cell Types and Their Computational Classification"
collection: publications
permalink: /publication/2020_10_05_SeqSeek
excerpt: 'We reveal spinal cord cell type organization, validate a combinatorial set of markers for in-tissue spatial gene expression analysis, and optimize the computational classification of spinal cord cell types based on transcriptomic data.'
date: 2020-07-01
venue: 'Under Review, eLife'
citation: Daniel E. Russ<sup>\*</sup>, Ryan B. Patterson-Cross<sup>\*</sup>, Li Li, Stephanie C. Koch, Kaya J.E. Matson, Ariel J. Levine (2020). &quot;Selecting single cell clustering parameter values with sub-sampling robustness metrics.&quot; <i>Under Review, eLife</i>.'
---


## ABSTRACT 

Single cell sequencing is transforming many fields of science but the vast amount of data it creates has the potential to both illuminate and obscure underlying biology. To harness the exciting potential of single cell data for the study of the mouse spinal cord, we have created a harmonized atlas of spinal cord transcriptomic cell types that unifies six independent and disparate studies into one common analysis. With the power of this large and diverse dataset, we reveal spinal cord cell type organization, validate a combinatorial set of markers for in-tissue spatial gene expression analysis, and optimize the computational classification of spinal cord cell types based on transcriptomic data. This work provides a comprehensive resource with unprecedented resolution of spinal cord cell types and charts a path forward for how to utilize transcriptomic data to expand our knowledge of spinal cord biology.

_\*_ These authors contributed equally to this work
